### PR TITLE
hp.support.assistant version 12.19.53.13

### PR DESCRIPTION
**Pull request by tmhakim**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16222)

### CODE REVIEW

File: manifests/h/hp/support/assistant/12.19.53.13/hp.support.assistant.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: hp.support.assistant
+PackageVersion: 12.19.53.13
+PackageLocale: en-US
+Publisher: HP Computers
+PackageName: HP Support Assistant
+License: Copyright c 2020-2021 Hewlett Packard
+ShortDescription: HP Support Assistant
```
Comment (mddtest) [inline manifests/h/hp/support/assistant/12.19.53.13/hp.support.assistant.locale.en-US.yaml:9]: ShortDescription: HP Support Assistant is a free utility on Hewlett Packard computers released after 2012 that helps avoid and resolve problems with the computer using updates and self-help options.

Comment (mddtest) [reply manifests/h/hp/support/assistant/12.19.53.13/hp.support.assistant.locale.en-US.yaml:9]: Path names should be capitalized as well  `HP/Support/Assistant`

Comment (denelon) [reply manifests/h/hp/support/assistant/12.19.53.13/hp.support.assistant.locale.en-US.yaml:9]: @mddtest thank you for helping improve PRs.

Did you know you can use the "[suggest](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)" syntax to make it easier for the author to just hit "accept changes"

File: manifests/h/hp/support/assistant/12.19.53.13/hp.support.assistant.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: hp.support.assistant
+PackageVersion: 12.19.53.13
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://ftp.ext.hp.com/pub/softlib/software13/HPSA/HPSupportSolutionsFramework-12.19.53.13.exe
+  InstallerSha256: 0F6BF99EF4DAD41F159CBFD37BE395BFD4BED0AF0978F1851BADECB554F9918D
+ManifestType: installer
```
Comment (mddtest) [inline manifests/h/hp/support/assistant/12.19.53.13/hp.support.assistant.installer.yaml:10]:
```suggestion
ManifestType: installer
InstallerSwitches:
    Custom: /SUPPRESSMSGBOXES /NORESTART
    Silent: /VERYSILENT
    SilentWithProgress: /VERYSILENT
InstallModes:
  - "interactive"
  - "silent"
  - "silentWithProgress"
```

**Pull request closed without merging**